butane-installation-configuration.yaml

```diff
@@ -1,32 +0,0 @@
-variant: openshift
```

Comment:
Mistake?

Reply:
No, it had a diff so I assumed it's generated